src/Microsoft.AspNet.Routing/ServiceCollectionExtensions.cs

```diff
@@ -17,7 +17,7 @@ public static class ServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The services available in the application.</param>
         /// <param name="setupAction">An action to configure the <see cref="RouteOptions"/>.</param>
-        public static void ConfigureRouteOptions(
+        public static void ConfigureRoute(
```

Comment:
looks like there is another issue which says `ConfigureRouting` (https://github.com/aspnet/Routing/issues/159)